storybook/src/admin-rte/field/RequiredField.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button, Field, FieldSet, TextField } from "@comet/admin";
+import { createFinalFormRte } from "@comet/admin-rte";
+import { Box, Grid } from "@mui/material";
+import { convertFromRaw } from "draft-js";
+import { Form } from "react-final-form";
+import { FormattedMessage } from "react-intl";
+
+const { RteField } = createFinalFormRte();
+
+export default {
+    title: "@comet/admin-rte/field",
+};
+
+type FormValues = {
+    rteContent: string;
+    rteContentRequired: string;
+    plainText: string;
+    plainTextRequired: string;
+};
+
+const richTextRequiredValidator = (value: string) => {
```

Comment:
@johnnyomair @nsams how do you think we should handle the required-validation for RTE fields? 
See the suggestions in the PR description 🙂 

Or is there already a solution that I missed that simply needs to be better documented? 😅

Reply:
> I believe we can go with suggestions 1 and 2, or both. 

A combination of 1 and 2 might make most sense. 
I'll create a separate PR for this 🙂

Reply:
> createRichTextBlock changes format and parse to remove the stringification. 🤯

Aaahh, good catch 👍🏼 
But the Block doesn't even use the `createFinalFormRte` and has no way to set `required` as far as I can see so this should be irrelevant here.